holoviews/operation/datashader.py

```diff
@@ -651,3 +658,32 @@ def _process(self, element, key=None):
         return element.clone(new_data)
 
 
+def split_dataframe(path_df):
+    """
+    Splits a dataframe of paths separated by NaNs into individual
+    dataframes.
+    """
+    splits = np.where(path_df.iloc[:, 0].isnull())[0]+1
+    return [df for df in np.split(path_df, splits) if len(df) > 1]
+
+
+class bundle_graph(Operation, hammer_bundle):
```

Comment:
Another graph operation ... but this time depending on datashader. Not sure where these should go...